docroot/themes/humsci/su_humsci_theme/scss/components/molecules/hover-menu.scss

```diff
@@ -142,6 +142,13 @@
                 margin-right: 0;
               }
             }
+            // Last two submenus should hug the right.
+            &:last-of-type,
+            &:nth-last-of-type(2) {
+              ul.decanter-nav-submenu {
+                right: 0;
```

Comment:
maybe we only do this when the screen is small enough? i'm not sure if theres a pure css way...  might need some javascript

Reply:
@pookmish Yes, as Joe and I were discussing yesterday, there is a small window where we have wrapping of the menu -- at that point, those last two get stranded to the right.  So, in the next sprint, I have a simple js idea to open a ticket about.